website/docs/d/db2_instance.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Db2 SaaS"
+layout: "ibm"
+page_title: "IBM : ibm_db2"
+description: |-
+  Get Information about IBM Db2 SaaS instance.
+---
+
+# ibm_db2
+
+Retrieve information about an existing [IBM Cloud Db2 SaaS Instance](https://cloud.ibm.com/docs/cloud-databases).
```

Comment:
Change this to "Retrieve information about an existing [IBM Db2 SaaS Instance](https://cloud.ibm.com/docs/Db2onCloud).